test/parseMessageDefinition.js

```diff
@@ -0,0 +1,265 @@
+// Copyright (c) 2018-present, GM Cruise LLC
+
+// This source code is licensed under the Apache License, Version 2.0,
+// found in the LICENSE file in the root directory of this source tree.
+// You may not use this file except in compliance with the License.
+
+import { expect } from "chai";
+import { parseMessageDefinition } from "../lib/parseMessageDefinition";
+
+describe("MessageDefinitionParser", () => {
```

Comment:
Forgot to rename this to `parseMessageDefinition`. 😄

Reply:
fixed